### ISSUE TITLE
(deferred) Starhunter configuration

### ISSUE THREAD

**Issue by vcarl** [Euno-HQ/discord#97]
(this project has been deferred as not a priority for the time being. the feature is valuable but no longer closely aligned with the immediate-term goals; it's disconnected from the rest of the featureset, and deserves more attention than I can give it right now)

The Starhunter feature shouldn't begin collecting data until it's been configured, and should collect a minimal set of data asked of it.

This should be able to be done via either the web or within Discord; an MVP of this might be easiest to do within Discord, but a full product should have it as part of an onboarding flow.

**Comment by vcarl** [comment]
Also noting: This will need to have a single responsible individual who kicks the process off. See https://github.com/reactiflux/mod-bot/issues/94#issuecomment-2555993097.
